README.md

```diff
@@ -105,6 +105,38 @@ React.render(<App person={person} />, document.body)
 person.name = "Mike" // will cause the Observer region to re-render
 ```
 
+Example2:
+
+```javascript
+class App extends React.Component {
+  render() {
+     return (
+          <Provider h="hello" w="world">
```

Comment:
Are you sure you can pass a string as a store prop? I am not aware that it would turn to observable automagically here. In that case, it's kinda misleading example.